src/components/BarChart.js

```diff
@@ -10,8 +10,11 @@ const Bar = React.createClass({
   propTypes: {
     animateOnHover: React.PropTypes.bool,
     animationDuration: React.PropTypes.number,
+    hasNegative: React.PropTypes.bool,
+    hasPositive: React.PropTypes.bool,
```

Comment:
I don't think think this is necessary. It's being computed and stored on the state down below.